python/sitecustomize.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import time
+
+
+# Avoiding additional imports by defining PY2 manually
+PY2 = sys.version_info[0] == 2
+
+# Define some debug logging routines to help sort out things when this
```

Comment:
Is this meant to be a future TODO? If so we should probably add the TODO tag in front of it. Otherwise, maybe just delete it as I don't think it's necessary.

Reply:
That comment is referring to the `def log_message(text, *args, **kwargs):` function, it's been there for a long time. I just moved it down to be next to the actual code it touches.